clap_complete/tests/testsuite/dynamic.rs

```diff
@@ -685,6 +724,36 @@ pos_c"
     );
 }
 
+fn suggest_custom_valuehint_panic() {
+    #[derive(Debug, Clone)]
+    struct PanicHint {}
+
+    impl clap::builder::CustomValueHint for PanicHint {
+        fn custom_hint(&self) -> Option<Vec<std::ffi::OsString>> {
+            panic!()
```

Comment:
Whats the purpose of this?

Reply:
I hope that the custom code does not affect other completion results, especially when panic occurs in the custom code.

Reply:
I don't see a reason to be making a decision on panic on behalf of the user.  We don't do this for other user-provided traits, like `TypedValueParser`